CoinDock-app/src/Screens/Account/EditPopup.js

```diff
@@ -82,60 +101,80 @@ const EditPopup = () => {
-          disabled={!isValid}
-        >
-          <div className="d-flex justify-content-between">
-            <h4>Account settings</h4>
-            <RiCloseLine
-              size="30px"
-              cursor="pointer"
-              onClick={() => handleSetTrigger(false)}
-            />
-          </div>
-          {type === "email" ? (
-            <Email
-              name="email"
-              formErrors={formErrors}
-              email={currentFieldValue}
-            />
-          ) : type === "changePassword" ? (
-            <Password
-              name="password"
-              formErrors={formErrors}
-              placeholder="Enter your password"
-              label="Password"
-            />
-          ) : type === "name" ? (
-            <div>
-              <Name
-                name="firstname"
-                placeholder="Enter First Name"
-                label="First Name"
-                currentFieldValue={first}
-                formErrors={formErrors}
-              />
-              <Name
-                name="lastname"
-                placeholder="Enter Last Name"
-                label="Last Name"
-                currentFieldValue={last}
-                formErrors={formErrors}
-              />
-            </div>
-          ) : (
-            ""
-          )}
-        </Popup>
-      </form>
+return(
+<div>
+<form  onInput ={handleChanges} >
+  <Popup
+    trigger={open}
+    setTrigger={handleSetTrigger}
+    buttonLable="Done"
+    buttonOnclick={handleSubmit}
+    disabled={!isValid}
+    >
+    <div className="d-flex justify-content-between">
+      <h4>Account settings</h4>
+      <RiCloseLine
+        size="30px"
+        cursor="pointer"
+        onClick={() => handleSetTrigger(false)}
+      />
     </div>
-  );
-};
+    { type === 'email' ? 
+    <Email
+      name="email"
+      formErrors={formErrors}
+      value={formValues.email}
+      />:
+     type === 'changePassword' ? 
+    <Password 
+      name="password"
+      formErrors={formErrors}
+      placeholder="Enter your password"
+      label="Password"
+    /> :
+    type == 'dateofbirth' ? 
+    <DatePick
+       name="date"
+       value={formValues.date}
+       onChange={handleChanges}
+       formErrors={formErrors}
+    />:
+    type == 'country' ?
+    <Select
+      name="country"
+      label="Country"
+      value={formValues.country}
+      options={[
+      {label:""},
+      { label: value.country },
+      { label: "India", value: 1 },
+      { label: "Pakistan", value: 2 },
+       ]}
+       formErrors={formErrors}
+      />:
+    type == 'name'?
+    <div>
+   <Name
+      name="firstname"
+      placeholder="Enter First Name"
+      label="First Name"
+      value={formValues.firstname}
+      formErrors={formErrors}
+      /> 
+    <Name
+      name="lastname"
+      placeholder="Enter Last Name"
+      label="Last Name"
+      value={formValues.lastname}
+      formErrors={formErrors}
+      />
+    </div>:''
```

Comment:
I think it is better to display null or nothing instead of empty string.

Reply:
Resolved